doc/rtd/development/packaging.rst

```diff
@@ -0,0 +1,93 @@
+.. _downstream_packaging:
+
+Downstream packaging
+********************
+
+This page is intended to support operating system packagers of ``cloud-init``
+and is not intended for other audiences to generate their own custom cloud-init
+packages. One should first look to official downstream cloud images containing
+pre-built packages of ``cloud-init`` for your preferred operating system.
```

Comment:
The second sentence is not intended for the audience that the first sentence says that this page is for. This is either self-contradictory or redundant or both, depending on how you look at it. In any case this is confusing and unnecessary.

We can drop the second sentence, since that content exists elsewhere in the documentation and isn't necessary for downstream packagers.